upload/src/addons/SV/WarningImprovements/Finder/WarningCategory.php

```diff
@@ -17,6 +17,7 @@ public function inParentWarningCategory(\SV\WarningImprovements\Entity\WarningCa
     public function rootOnly()
     {
         $this->where('parent_category_id', '=', 0);
+        $this->where('parent_category_id', '!=', null);
```

Comment:
If you haven't run the migration steps, there isn't going to be anything with a null value so this will look to work. 

I believe the correct version should be;
```php
    public function rootOnly()
    {
        $this->where('parent_category_id', '=', null);

        return $this;
    }
```

Reply:
Categories which act as parent are having 0 as parent_category_d right now

https://github.com/ticktackk/XenForo2-WarningImprovements/blob/master/upload/src/addons/SV/WarningImprovements/XF/Admin/Controller/Warning.php#L196

Reply:
Chuck the following after the filter statement?
```php
        if (!$input['parent_category_id'])
        {
            $input['parent_category_id'] = null;
        }
```

Reply:
FYI, merged your most recent commit